core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php

```diff
@@ -306,6 +321,37 @@ private function isGranted(string $actionClass, string $table, DataContainerOper
             default => throw new \InvalidArgumentException(\sprintf('Invalid action class "%s".', $actionClass)),
         };
 
-        return $this->security->isGranted(ContaoCorePermissions::DC_PREFIX.$table, $subject);
+        // TODO: class always exists when we require at least Symfony 7.3+
+        $accessDecision = class_exists(AccessDecision::class) ? new AccessDecision() : null;
+
+        $isGranted = $this->authorizationChecker->isGranted(ContaoCorePermissions::DC_PREFIX.$table, $subject, $accessDecision);
+
+        return $accessDecision ?? $isGranted;
```

Comment:
Would not need to use the checker if it's not null, right?

```suggestion
        if (null === $accessDecision) {
            return $this->authorizationChecker->isGranted(ContaoCorePermissions::DC_PREFIX.$table, $subject, $accessDecision);
        }

        return $accessDecision;
```

Reply:
No that's wrong. The `$accessDecision` object is passed to the `isGranted` method and filled with the result. But if the `AccessDecision` class does not exist (Symfony < 7.3), we return the boolean result instead.